src/pp-annotator/editor/EditorContent.scss

```diff
@@ -2,9 +2,10 @@
 
   .header-buttons {
     display: flex;
+    justify-content: space-between;
 
-    :not(:first-child):not(:last-child) {
-      margin: 0 12px;
+    button {
+      width: 32%;
```

Comment:
`flex-grow: 1` should be enough, I think?
setting fixed width inside `flex` containers is usually a "last resort" solution 😛